torchci/lib/bot/triggerInductorTestsBot.ts

```diff
@@ -0,0 +1,66 @@
+import { Context, Probot } from "probot";
+
+export default function triggerInductorTestsBot(app: Probot): void {
+  const preapprovedUsers = ["pytorchbot", "PaliC"]; // List of preapproved users
+  //   const tritonRepo = "triton-lang/triton"; // uncomment once this is good enough for triton
+  const tritonRepo = "triton-lang-test/triton"; // delete once this is good enough for triton
+  const preapprovedRepos = ["malfet/deleteme", tritonRepo]; // List of preapproved orgs/repos
+  //   const preapprovedRepos = ["malfet/deleteme", "triton-lang/triton"];// uncomment once this is good enough for triton and delete line above
+
+  app.on(
+    ["issue_comment.created"],
+    async (ctx: Context<"issue_comment.created">) => {
+      const commentBody = ctx.payload.comment.body.toLowerCase();
+      const commenter = ctx.payload.comment.user.login;
+      const orgRepo = `${ctx.payload.repository.owner.login}/${ctx.payload.repository.name}`;
+      if (
+        commentBody.includes("trigger inductor tests") &&
```

Comment:
Nit: I think we can just call it `@pytorchbot run pytorch tests` or something more generic.  Maybe we want to run more than just inductor tests in the future